src/ConcreteOperations/minkowski_sum.jl

```diff
@@ -235,9 +237,13 @@ A `Zonotope` corresponding to the concrete Minkowski sum of `Z1` and `Z2`.
 ### Algorithm
 
 The resulting zonotope is obtained by summing up the centers and concatenating
-the generators of `Z1` and `Z2`.
+the generators of `Z1` and `Z2`. If the flag `remove_zero_generators` is set to
+`true`, the zero generators (i.e. the generators which have all its components
+equal to zero) of `Z1 ⊕ Z2` are removed.
 """
-function minkowski_sum(Z1::AbstractZonotope{N},
-                       Z2::AbstractZonotope{N}) where {N<:Real}
-    return Zonotope(center(Z1) + center(Z2), [genmat(Z1) genmat(Z2)])
+function minkowski_sum(Z1::AbstractZonotope{N}, Z2::AbstractZonotope{N};
+                       remove_zero_generators::Bool=true) where {N<:Real}
+    cnew = center(Z1) + center(Z2)
+    Gnew = hcat(genmat(Z1), genmat(Z2))
+    return Zonotope(cnew, Gnew, remove_zero_generators=remove_zero_generators)
```

Comment:
```suggestion
    return Zonotope(cnew, Gnew, remove_zero_generators=false)
```

Reply:
Ok.. i disagree here. We may prefer to change this in the future, i know, but for now and as i said in the other issue, i prefer to use the constructor's defaults ( = keep the behavior in master).

Reply:
Well, then we seem to disagree.

Reply:
True, but we agree that the option should be added. Should we merge this PR?

Reply:
> True, but we agree that the option should be added.

No, I do not see a reason to add the option.